server/fusor.gemspec

```diff
@@ -13,4 +13,6 @@ Gem::Specification.new do |s|
   s.files      = Dir["{app,config,db,lib}/**/*"] + ["LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
+  s.add_dependency "active_model_serializers", '~> 0.9'
+
```

Comment:
Is there no way to accomplish this with rabl? We already are using one serialization library feels weird to add another for not much gain.